src/graphql/mirror.js

```diff
@@ -728,6 +740,68 @@ export class Mirror {
     }
   }
 
+  /**
+   * Save the request query and query parameters to the database.
+   */
+
+  async _log_request(
+    body: Queries.Body,
+    variables: {+[string]: any}
+  ): Promise<number> {
```

Comment:
Consider defining an internal type alias `type NetworkLogId = number`
for readability, like the existing `type UpdateId`? It’s not obvious
from the type signature what number the `Promise<number>` represents.

Reply:
updated